app/assets/stylesheets/components/_index.scss

```diff
@@ -6,3 +6,14 @@
 @import "banner";
 @import "buttons";
 @import "tags";
+@import "offer_card";
+@import "offers_banner";
+
+a {
```

Comment:
I'm not sure if 'global styles' for all pages are written directly in the index file. 

Maybe a partial like "_global_styles.scss" . Fine for now but let's check later what is best practice here.